coreapps-web-2.2/src/test/java/org/openmrs/module/coreapps/diagnosisCompatibility/DiagnosisServiceCompatibility2_2Test.java

```diff
@@ -0,0 +1,30 @@
+package org.openmrs.module.coreapps.diagnosisCompatibility;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.Encounter;
+import org.openmrs.api.EncounterService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+
+@Component
+
+public class DiagnosisServiceCompatibility2_2Test {
+	
+	@Autowired
+	@Qualifier("encounterService")
+	EncounterService encounterService;
+	
+	/**
+	 * @see DiagnosisServiceCompatibility2_2#getDiagnoses(Encounter)
+	 */
+	
+	@Test
+	public void getDiagnoses_shouldGetDiagnosesOfTheProvidedEncounter() {
+	Encounter encounter = encounterService.getEncounter(3);
```

Comment:
@ibacher   hi ian,so for this i have made changes in the core as seen here,https://github.com/openmrs/openmrs-core/pull/3682   at this line 26,i simply wanted to first check whether my submodule loads the openmrs api well,it looks like some thing is still missing since i get a null exception, https://pastebin.com/DxTuit8A a gentle request to help me look through my  coreapps-web-2.2/pom.xml  if i am missing out some thing cc @dkayiwa @mks-d